frontend/src/domains/plays/PlayDetails/PlayDetails.js

```diff
@@ -1,41 +1,65 @@
 import React from 'react'
 import styled from 'styled-components'
 
-import { imageOf } from '../../../services/playsClient'
 import { getLocaleDate } from '../../../utilities/getLocaleDate'
+import { imageOf } from '../../../services/playsClient'
 import Icon from '../../../utilities/Icon' 
+import PlayStarRatingEdit from '../../../components/inputs/Input'
+import PlayingTimeEdit from '../../../components/inputs/Input'
```

Comment:
Hast du in dem Folder 'inputs' mehrere Input Komponenten? Dann könntest du die Komponenten selbst auch entsprechend benennen, wie zb TextInput, SelectInput etc...

Reply:
Ja, die haben alle einen definierten Namen. Ich bin gerade dabei, die Komponente umzubenennen. Da aber mein Auto-Update gerade nicht funktioniert, lasse ich das erst einmal, leider.